backend/coins/eth/account.go

```diff
@@ -610,15 +605,41 @@ func (account *Account) newTx(args *accounts.TxProposalArgs) (*TxProposal, error
 			}
 		}
 	}
-	tx := types.NewTransaction(account.nextNonce,
-		*message.To,
-		message.Value, gasLimit, suggestedGasPrice, message.Data)
+
+	keystore, err := account.Config().ConnectKeystore()
```

Comment:
hmmmm I think this is called when modifying anything in the Send form, e.g. when pasting a recipient address, changing the amount, etc.

In watch-only mode, this means as soon as you do anything in the Send form, a 'Connect your BitBox02' prompt will appear, right? that would be different to how it was before and how it is in bitcoin, where the prompt only happens when one clicks to sign/send.

Reply:
hmm, yeah, good catch, thoughts how to handle it? 
seems strange to have the send screen available in watch only mode? Maybe the send button should only be enabled when the device is connected? 
Or do you want to handle it here in the send screen instead?

Reply:
As discussed in private chat - we will prompt to connect the keystore when going to the send scree as you suggested, but only for ETH accounts. This is implemented here:

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2456/